src/Algebra/Properties/Monoid/Sum.agda

```diff
@@ -70,3 +71,21 @@ sum-replicate-idem idem n = trans (sum-replicate n) (×-idem idem n)
 sum-replicate-zero : ∀ n → sum {n} (replicate 0#) ≈ 0#
 sum-replicate-zero zero    = refl
 sum-replicate-zero (suc n) = sum-replicate-idem (+-identityˡ 0#) (suc n)
+
+------------------------------------------------------------------------
+-- When summing over a `Vector`, we can pull out last element
+
+sum-init : ∀ {n} (t : Vector Carrier (suc n)) → sum t ≈ sum (init t) + last t
```

Comment:
A better name might be `sum-init-last`?

Reply:
Yes, I had been taking the name directly from #1287 . Suggestion is better!